test/Splunk.OpenTelemetry.AutoInstrumentation.IntegrationTests/BuildTests.cs

```diff
@@ -64,12 +64,15 @@ private static string GetSystemName()
 
         if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
         {
+            var processArchitecture = RuntimeInformation.ProcessArchitecture;
```

Comment:
reuse `EnvironmentTools.GetPlatform()`, EnvironmentTools should be the source of truth for naming.

Reply:
f6e39d470cc62bc4cb0967338e96679406ca6a9d